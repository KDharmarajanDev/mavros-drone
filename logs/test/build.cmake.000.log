Not searching for unused variables given on the command line.
[31m[1mCMake Error[0m at /home/karthikdharmarajan/catkin_ws/src/test/CMakeLists.txt:69:
  Parse error.  Expected "(", got newline with text "

  ".


[36m--[0m Configuring incomplete, errors occurred!
