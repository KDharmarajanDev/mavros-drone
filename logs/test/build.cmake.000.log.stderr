[31m[1mCMake Error[0m at /home/karthikdharmarajan/catkin_ws/src/test/CMakeLists.txt:69:
  Parse error.  Expected "(", got newline with text "

  ".


